tensorflow/core/kernels/linalg/lu_op_gpu.cu.cc

```diff
@@ -182,9 +182,16 @@ class LuOpGpu : public AsyncOpKernel {
       auto packed_triangular_factors_ptrs = solver->GetScratchSpace<uint8>(
           sizeof(Scalar*) * batch_size, "packed_triangular_factors_ptrs",
           /* on_host */ true);
+#if GOOGLE_CUDA
```

Comment:
Does the code below need to be duplicated?

Reply:
@cheshire removed duplicated code.